test/Component.js

```diff
@@ -125,155 +125,34 @@ describe('Component', () => {
 
 	it('should render a generator component', () => {
 		const target = document.createElement('div')
-		const Primary = function * (props) { yield 1, yield 2 }
+		const Primary = function* (props) { yield 1, yield 2 }
 
 		render(h(Primary), target, (current) => {
 			assert.html(current, '12')
 		})
 	})
 
-	it('should render and update an async component', (done) => {
+	it('should update pure component', () => {
```

Comment:
This does not differentiate `update` and `replace` (same with the test below).
The `<h1>` node in the DOM itself should be asserted for equality between renders.

```js
el = target.firstChild
render()
assert(el == target.firstChild)
```